lectures/01.intro.md

```diff
@@ -333,7 +378,7 @@ The game is constantly played, tested, changed
 
 # Why make games as business?
 
-It's ~~not~~ about the money
+Games do ~~not~~ earn a lot

```

Comment:
I prefer the old statement but won't insist on changing it

Reply:
I think the previous version can be interpreted as "GameDev is ONLY about money".
The change tries to convey the information that there are a lot of money in Games, without specifying any agenda.

Reply:
fair enough